cfme/common/vm.py

```diff
@@ -717,6 +725,69 @@ def set_retirement_date(self, when, warn=None):
             sel.click(form_buttons.save)
 
 
+    def equal_drift_results(self, row_text, section, *indexes):
+        """ Compares drift analysis results of a row specified by it's title text
+
+        Args:
+            row_text: Title text of the row to compare
+            section: Accordion section where the change happened; this section must be activated
```

Comment:
I think you should change the language here - when I read this, I thought I would have to activate the accordion tree before calling the function. I'd suggest stating the section will be activated, since the function does that.